### PR TITLE
[YUNIKORN-2890] fix broken table in "Labels and Annotations in YuniKorn" page

### PR DESCRIPTION
**Pull request by ryankert01**
### What is this PR for?
refer to https://yunikorn.apache.org/docs/user_guide/labels_and_annotations_in_yunikorn#labels-in-yunikorn

Before:
截圖 2024-09-26 凌晨2 15 28

After
截圖 2024-09-26 凌晨2 16 26

### What type of PR is it?
* [x] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-2890

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

File: docs/user_guide/labels_and_annotations_in_yunikorn.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ The non-canonical (legacy) representations will not be deprecated and will conti
...
 | `yunikorn.apache.org/app-id` | Associates this pod with an application. This is the canonical representation and it is recommended to use it.<br/><br/>The priority of application ID is determined in the following order: <ol><li>Label `yunikorn.apache.org/app-id`</li><li>Annotation `yunikorn.apache.org/app-id`</li><li>Label `applicationId`</li><li>Label `spark-app-selector`</li></ol>**Note**: Pods with inconsistent application metadata will be rejected in 1.7.0.                                                                                                                                                                                                                                                                                      |     |
```
Comment (chenyulin0719) [inline docs/user_guide/labels_and_annotations_in_yunikorn.md:38]: Could you also remove the last "|"?

**Review by chenyulin0719 — CHANGES_REQUESTED** [review]
Thanks for pointing out the issue, we should also update the versioned page of 1.6.0.

File: docs/user_guide/labels_and_annotations_in_yunikorn.md (hunk 1 above)
Comment (ryankert01) [reply docs/user_guide/labels_and_annotations_in_yunikorn.md:38]: Good catch! Will do.

Comment (ryankert01) [reply docs/user_guide/labels_and_annotations_in_yunikorn.md]: Done.

**Review by chenyulin0719 — CHANGES_REQUESTED** [review]
+1 LTGM

**Review by chenyulin0719 — APPROVED** [review]

**Pull request closed without merging**